### DISCUSSION TITLE
Update Url

### DISCUSSION THREAD

**Discussion by LeonWehrhahn** [jonataslaw/getx#1256] [Ideas]
**Proposal:**

It would be really useful if there was a function like `Get.updateUrl("")`, that would update the url, without sending the user anywhere.

**Reason:**

When a user is for example on a search page. It would be great, if we could update the url on every search he enters. That way, when the user shares a website link of that search page to someone, we will be able to show him the same search results, because of the updated parameters in the url.

**Comment by jonataslaw** [comment]
Moved to discussions
